mkdocs.yml

```diff
@@ -2,18 +2,30 @@ site_name: Uvicorn
 site_description: The lightning-fast ASGI server.
 
 theme:
-    name: 'material'
-    custom_dir: docs/overrides
+  name: "material"
+  custom_dir: docs/overrides
+  palette:
+    - scheme: "default"
+      media: "(prefers-color-scheme: light)"
+      toggle:
+        icon: "material/lightbulb"
+        name: "Switch to dark mode"
+    - scheme: "slate"
+      media: "(prefers-color-scheme: dark)"
+      primary: "blue"
+      toggle:
+        icon: "material/lightbulb-outline"
+        name: "Switch to light mode"
 
 repo_name: encode/uvicorn
 repo_url: https://github.com/encode/uvicorn
 edit_uri: ""
```

Comment:
Here it was using double quotes, so I've set everything to use double quotes instead of single.